public/locales/ar/profile.json

```diff
@@ -0,0 +1,24 @@
+{
+    "10ticketsRemaining": "تبقى عشرة تذاكر",
+    "3cardsAdded": "تم اضافة ثلاثة بطاقات",
```

Comment:
It could be better to make these two keys trasnlation as reusable.
Since it might be 3, 10, 4, ... whatever.
So, you can use interpolation to inject (count) variable here.
for example:
`"ticketRemaining": "تبقى {{ count }} تذاكر",`

You can read more about this here:
[How To Pass a Variable to Your Translation String With i18next](https://betterprogramming.pub/how-to-pass-a-variable-to-your-translation-string-with-i18next-b07614c33514)